models/repo.go

```diff
@@ -126,18 +126,32 @@ func NewRepoContext() {
 	RemoveAllWithNotice("Clean up repository temporary data", filepath.Join(setting.AppDataPath, "tmp"))
 }
 
+// GitServiceType represents a git service
+type GitServiceType int
+
+// enumerate all GitServiceType
+const (
+	NotMigrated     GitServiceType = iota // 0 not migrated from external sites
```

Comment:
Changing this enum numbers will result in inconsistent values with already saved in database

Reply:
I could update the migrated repositories to the corrected type since we currently only support GithubService or PlainGitService(All non-github repositories could be considered as plaingitservice.)

Reply:
This has been implemented on `models/migrations` on recent changes.